cypress/features/regression/radioButtonGroupClassic.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Experimental RadioButtonGroup classic component
```

Comment:
the same here please, could we move also that features file to `cypress/features/regression/experimental/`